### PR TITLE
Adjust logging output

### PR DESCRIPTION
**Pull request by jmthomas**

### CODE REVIEW

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue [hunk 1]
```diff
@@ -29,7 +29,7 @@
           show-size
           accept=".gem"
           class="mx-2"
-          label="Click to select plugin .gem file to install"
+          label="Click to install new plugin .gem (NOT upgrade)"
```
Comment (jmthomas) [inline openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue:32]: I wanted this to be clearer that this is for installing new plugins. I think we're still getting people clicking here for upgrade which leads to the "already exists" error.

File: openc3/python/openc3/microservices/microservice.py [hunk 2]
```diff
@@ -194,7 +194,7 @@ def shutdown(self):
         if self.temp_dir is not None:
             self.temp_dir.cleanup()
         self.metric.shutdown()
-        self.logger.info(f"Shutting down microservice complete: {self.name}")
+        self.logger.debug(f"Shutting down microservice complete: {self.name}")
```
Comment (ryanmelt) [inline openc3/python/openc3/microservices/microservice.py:197]: Restore to info

File: openc3/lib/openc3/operators/operator.rb [hunk 3]
```diff
@@ -314,38 +314,38 @@ def shutdown_processes(processes)
...
-        processes.each { |name, p| p.hard_stop }
+        Logger.debug("Commanding hard stops...")
+        processes.each { |_name, p| p.hard_stop }
       end
     end
 
     def shutdown
       @shutdown = true
       @mutex.synchronize do
-        Logger.info("Shutting down processes...")
```
Comment (ryanmelt) [inline openc3/lib/openc3/operators/operator.rb:346]: Restore to info

File: openc3/lib/openc3/operators/operator.rb [hunk 4]
```diff
@@ -314,38 +314,38 @@ def shutdown_processes(processes)
...
+        Logger.debug("Shutting down processes...")
         shutdown_processes(@processes)
-        Logger.info("Shutting down processes complete")
+        Logger.debug("Shutting down processes complete")
```
Comment (ryanmelt) [inline openc3/lib/openc3/operators/operator.rb:348]: Restore to info

File: openc3/lib/openc3/operators/operator.rb [hunk 5]
```diff
@@ -174,7 +174,7 @@ def hard_stop
       if @process and !@process.exited?
         # Redis may be down at this point so just catch any Logger errors
         begin
-          Logger.info("Hard shutting down process: #{cmd_line()}", scope: @scope)
+          Logger.debug("Hard shutting down process: #{cmd_line()}", scope: @scope)
```
Comment (ryanmelt) [inline openc3/lib/openc3/operators/operator.rb:177]: Restore to info

File: openc3/lib/openc3/operators/operator.rb [hunk 6]
```diff
@@ -161,7 +161,7 @@ def exit_code
 
     def soft_stop
       Thread.new do
-        Logger.info("Soft shutting down process: #{cmd_line()}", scope: @scope)
+        Logger.debug("Soft shutting down process: #{cmd_line()}", scope: @scope)
```
Comment (ryanmelt) [inline openc3/lib/openc3/operators/operator.rb:164]: Restore to info

File: openc3/lib/openc3/microservices/microservice.rb [hunk 7]
```diff
@@ -208,7 +208,7 @@ def shutdown
       MicroserviceStatusModel.set(as_json(:allow_nan => true), scope: @scope)
       FileUtils.remove_entry(@temp_dir) if File.exist?(@temp_dir)
       @metric.shutdown
-      @logger.info("Shutting down microservice complete: #{@name}")
+      @logger.debug("Shutting down microservice complete: #{@name}")
```
Comment (ryanmelt) [inline openc3/lib/openc3/microservices/microservice.rb:211]: Restore to info

File: openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue (hunk 1 above)
Comment (ryanmelt) [reply openc3-cosmos-init/plugins/packages/openc3-tool-common/src/tools/admin/tabs/PluginsTab.vue:32]: The real fix for this is a new "Are you sure" prompt that lets the user know that the plugin they are trying to install has already been installed at least once.   This info could be returned in the results from install_phase1

**Review by ryanmelt — CHANGES_REQUESTED** [review]

**Comment by jmthomas** [comment]
Added #1398 to address issue about installing with "Are you sure" prompt.

**Pull request merged**